Microsoft.Toolkit.Uwp.UI.Controls/WrapPanel/WrapPanel.Data.cs

```diff
@@ -19,20 +20,18 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls.WrapPanel
     /// </summary>
     public partial class WrapPanel
     {
-        internal class UvMeasure
+        [System.Diagnostics.DebuggerDisplay("U = {U} V = {V}")]
+        private struct UvMeasure
         {
-            internal double U { get; set; }
-
-            internal double V { get; set; }
+            private double u, v;
+            internal double U { get { return u; } set { u = Math.Floor(value); } }
 
-            public UvMeasure()
-            {
-                U = 0.0;
-                V = 0.0;
-            }
+            internal double V { get { return v; } set { v = Math.Floor(value); } }
 
             public UvMeasure(Orientation orientation, double width, double height)
```

Comment:
The default constructor was used to reinitialize U and V values, the default constructor was used in the wrapPanel class multiple times. I know that you're using a struct which has no a default constructor so either you go back to class or create a function to reset the values to zeroes which should be used in the wrapPanel.cs class.